Runtime/Scripts/EdgeEventsManager.cs

```diff
@@ -197,6 +233,14 @@ public void StartEdgeEvents(ConnectionDetails connectionDetails)
     {
       Logger.Log("Starting EdgeEvents");
       config = MobiledgeXIntegration.settings.edgeEventsConfig;
+      if (IsMainThread) // Input.location can be accessed only from UI Thread
+      {
+        StartCoroutine(locationService.UpdateEdgeEventsManagerLocation(Math.Min(config.locationConfig.updateIntervalSeconds, config.latencyConfig.updateIntervalSeconds)));
```

Comment:
What happens if Min == 0?

Reply:
Good point will remove this code block to right after the ConfigValidation so if either of them == 0 it would throw an exception.

Reply:
Fixed, Now if locationConfig.updateIntervalSeconds =0 or latencyConfig.updateIntervalSeconds =0, An error will be produced and EdgeEvents won't start.